README.md

```diff
@@ -2,11 +2,28 @@
 
 
 <!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Deployment](#deployment)
```

Comment:
- [ ] Great work on adding the table of contents but your Deployment link is not working :fearful: Kindly fix this link to make your readme descriptive enough :+1: